src/commands/checkIn/checkIn.js

```diff
@@ -138,3 +145,37 @@ const remove = async (interaction) => {
     }
 
 };
+
+const viewResponses = async (interaction) => {
+    await interaction.deferReply({ ephemeral: false });
```

Comment:
Is there a reason for the reply here not to be ephemeral? If it is an admin only command I think it would make sense for the interaction to be private

Reply:
The false was ephemeral more for testing purposes. The fact that it's an admin command doesn't matter. Especially since all data is being sent in dms.

Reply:
Alright, disregard that then